### PR TITLE
Update Getting Started section in Readme

### PR DESCRIPTION
**Pull request by florianb**
Remove outdated reference on `virtualenv` and Python 2.7

## Testing recommendations

Go through the documented steps.

## Checklist

- [x] Documentation is written and is accurate;
- [x] `make check` passes with no errors
- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,34 @@ You can view the latest rendered build of this content at:
...
-# Building this repo
+# Getting Started
 
-Install Python 2.7+ and pip. Then:
+## Prerequisites
+
+- Python 3.x
+- Pip
+
+### Preparing Prerequisites using [asdf](https://asdf-vm.com/#/)
```
Comment (wohali) [inline README.md:15]: I'd rather not give this "top billing."

Would you be willing to restore the virtualenv-based way of setting this up, and add this as the secondary way below it?

Comment (florianb) [reply README.md:15]: Of course @wohali  - i thought `virtualenv` wouldn't be used anymore since the `venv`-directory is missing. I will revert that.

Comment (florianb) [reply README.md:15]: @wohali nevermind - i made a mistake.

Comment (wohali) [reply README.md:15]: Here's what I'd recommend for updating the instructions to:

```
python3 -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
```

Comment (florianb) [reply README.md]: @wohali - done. Sorry for the noise.

**Pull request merged**